### PR TITLE
added isMK functionality

### PR DESCRIPTION
**Pull request by daancs**

### CODE REVIEW

File: prisma/seed.ts [hunk 1]
```diff
@@ -115,6 +115,16 @@ async function main() {
             accepted: true
         },
     })
+    await prisma.user.create({
+        data: {
+            name: 'Emil Lindblad',
```
Comment (emillindblad) [inline prisma/seed.ts:120]: Cool guy

File: src/hooks/useIsMK.tsx [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
 export const useIsMK = () => {
-  const [isMK, setIsMK] = useState(true)
-  useEffect(() => {
-    //TODO kod för att kolla att man är MK
-                },
-            []
-        )
-  return isMK
+  //TODO kod för att kolla att man är MK
```
Comment (emillindblad) [inline src/hooks/useIsMK.tsx]: Yikes, Looks like you forgot to remove a comment here :)

**Review by emillindblad — CHANGES_REQUESTED** [review]
Please se my comments :)

File: prisma/seed.ts (hunk 1 above)
Comment (daancs) [reply prisma/seed.ts:120]: no.

**Review by emillindblad — APPROVED** [review]
LGTM

**Pull request merged**